pkgs/by-name/op/opengist/package.nix

```diff
@@ -4,38 +4,37 @@
   buildNpmPackage,
   fetchFromGitHub,
   moreutils,
-  npm-lockfile-fix,
+  jd-diff-patch,
   jq,
   git,
 }:
 let
   # finalAttrs when 🥺 (buildGoModule does not support them)
   # https://github.com/NixOS/nixpkgs/issues/273815
-  version = "1.7.5";
+  version = "1.8.2";
   src = fetchFromGitHub {
     owner = "thomiceli";
     repo = "opengist";
     rev = "v${version}";
-    hash = "sha256-mZ4j9UWdKa3nygcRO5ceyONetkks3ZGWxvzD34eOXew=";
-
-    # follow https://github.com/thomiceli/opengist/pull/350 and remove here
-    postFetch = ''
-      ${lib.getExe npm-lockfile-fix} $out/package-lock.json
-    '';
+    hash = "sha256-mx1RPDvoewSypwRRKZBImTC6119Nlz3JD212hxjNbNA=";
   };
+  jd' = lib.getExe jd-diff-patch;
+  jq' = lib.getExe jq;
```

Comment:
Prime is usually used for overriden packages. Maybe just inline those or export PATH with it?

Reply:
Ok I will inline them.